meta-mender-core/recipes-bsp/u-boot/files/add_kconfig_option_with_depends.py

```diff
@@ -23,20 +23,30 @@ def term_parse(term_string):
     algebra = boolean.BooleanAlgebra()
     return algebra.parse(term_string)
 
-def term_reduce_to_first_alternative(term):
+def term_reduce_to_one_alternative(term, removal_terms):
```

Comment:
It is the first time I am looking at this piece of code, but the logic below seems correct. What I don't understand is why we named it `removal_terms` here but is `already_added` by the caller. I think the latter or something similar is better.

Reply:
The thinking was that the `term` set of functions is more related to logical algebra, not strictly related to Kconfig, so I switched the terminology to fit "independently" into that function. But maybe that just makes it more confusing. I'm ok with changing it if you think `already_added` is clearer.

Reply:
I see your point. It is not important, let's leave it as it is.